src/main/java/org/alliancegenome/curation_api/services/validation/dto/fms/VariantFmsDTOValidator.java

```diff
@@ -0,0 +1,325 @@
+			for (CrossReferenceFmsDTO xrefDto : dto.getCrossReferences()) {
+				ObjectResponse<List<CrossReference>> xrefResponse = crossReferenceFmsDtoValidator.validateCrossReferenceFmsDTO(xrefDto);
+				if (xrefResponse.hasErrors()) {
+					variantResponse.addErrorMessage("cross_references", xrefResponse.errorMessagesString());
+					break;
+				} else {
+					validatedXrefs.addAll(xrefResponse.getEntity());
+				}
+			}
+		}
+		List<CrossReference> xrefs = crossReferenceService.getUpdatedXrefList(validatedXrefs, variant.getCrossReferences());
+		if (variant.getCrossReferences() != null) {
+			variant.getCrossReferences().clear();
+		}
+		if (xrefs != null) {
+			if (variant.getCrossReferences() == null) {
+				variant.setCrossReferences(new ArrayList<>());
+			}
+			variant.getCrossReferences().addAll(xrefs);
+		}
+		
+		if (variant.getRelatedNotes() != null) {
+			variant.getRelatedNotes().clear();
+		}
+
+		List<Note> validatedNotes = new ArrayList<Note>();
+		List<String> noteIdentities = new ArrayList<String>();
+		Boolean allNotesValid = true;
+		if (CollectionUtils.isNotEmpty(dto.getNotes())) {
+			for (int ix = 0; ix < dto.getNotes().size(); ix++) {
+				ObjectResponse<Note> noteResponse = variantNoteFmsDtoValidator.validateVariantNoteFmsDTO(dto.getNotes().get(ix));
+				if (noteResponse.hasErrors()) {
+					allNotesValid = false;
+					variantResponse.addErrorMessages("notes", ix, noteResponse.getErrorMessages());
+				} else {
+					String noteIdentity = NoteIdentityHelper.variantNoteFmsDtoIdentity(dto.getNotes().get(ix));
+					if (!noteIdentities.contains(noteIdentity)) {
+						noteIdentities.add(noteIdentity);
+						validatedNotes.add(noteDAO.persist(noteResponse.getEntity()));
+					}
+				}
+			}
+		}
+		if (!allNotesValid) {
+			variantResponse.convertMapToErrorMessages("notes");
+		}
+		if (CollectionUtils.isNotEmpty(validatedNotes) && allNotesValid) {
+			if (variant.getRelatedNotes() == null) {
+				variant.setRelatedNotes(new ArrayList<>());
+			}
+			variant.getRelatedNotes().addAll(validatedNotes);
+		}
+		
+		if (variantResponse.hasErrors()) {
+			variantResponse.convertErrorMessagesToMap();
+			throw new ObjectValidationException(dto, variantResponse.errorMessagesString());
+		}
+		
+		variant = variantDAO.persist(variant);
+		if (variant != null) {
+			idsAdded.add(variant.getId());
+		}
+
+		return variant;
+	}
+	
+	@Transactional
+	public void validateCuratedVariantGenomicLocationAssociation(VariantFmsDTO dto, List<Long> idsAdded, Variant variant) throws ValidationException {
+
+		CuratedVariantGenomicLocationAssociation association = new CuratedVariantGenomicLocationAssociation();	
+		ObjectResponse<CuratedVariantGenomicLocationAssociation> cvglaResponse = new ObjectResponse<CuratedVariantGenomicLocationAssociation>();
+		AssemblyComponent chromosome = null;
+		
+		if (dto.getStart() == null) {
+			cvglaResponse.addErrorMessage("start", ValidationConstants.REQUIRED_MESSAGE);
+		} 
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.regexp.RegexpSinglelineJavaCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line must not have trailing whitespace
<!-- __reviewdog__:ChA0ZTJlZmYyMjVhYTYzZGFkEgpjaGVja3N0eWxl -->